### PR TITLE
London Class 7 - Laura M - HTML/CSS - Week 3

### PR DESCRIPTION
**Pull request by Laura-TM**
# Your Details

- Your Name: Laura M
- Your City: London
- Your Slack Name: Laura M

# Homework Details

- Module: HTML/CSS
- Week: 3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,10 +7,81 @@
...
 
 <body>
     <!-- Add your markup here -->
+    <header>
+        <div class="header-left">
+            <img src="design/cake-logo-transparent.png" alt="We teach cakes!" class="logoImage"/>
+        </div>
+        <div class="header-right"> 
+            <div class="mobile-menu">
+                <div class="menu"></div>
```
Comment (makanti) [inline index.html:25]: Nice touch using css to style your burger icon 👏 Considering this is just a decorative element I would choose to use a `<span>` tag.

File: index.html [hunk 2]
```diff
@@ -7,10 +7,81 @@
...
+            <div class="main-text">
+                <h4>Cake-ing!</h4>
+                <p>
+                    The term "cake" has a long history. The word itself is of Viking origin, from the Old Norse word "kaka". 
+                    Early cakes in England were also essentially bread. Sponge cakes, leavened with beaten eggs, originated during the Renaissance, possibly in Spain.
+                    At Cake, we hope to share with you different ways of making and baking these items, so you can do it yourself at home... or just order it from the professionals!
+                </p>
+            </div>
+            <div class="final-product">
+                <img src="design/cake-final-product.jpeg" alt="Finished cake" class="finished-cake"/>
```
Comment (makanti) [inline index.html:58]: Well done for using `alt` attributes on all image elements 💯

File: style.css [hunk 3]
```diff
@@ -1 +1,401 @@
 /*  Add your styling here */
+* {
+    font-family: 'Texturina', serif;
+}
+
+body {
+    width: 100%;
```
Comment (makanti) [inline style.css:7]: Why is that needed? 🤔

File: style.css [hunk 4]
```diff
@@ -1 +1,401 @@
...
+}
+
+header {
+    background-color: #d9e4f5;
+    background-image: linear-gradient(315deg, #d9e4f5 0%, #f5e3e6 74%);
```
Comment (makanti) [inline style.css:12]: Beautiful use of gradient 🌟

**Review by makanti — COMMENTED** [review]
Great effort Laura! Very nicely styled cake page, good use of flex and some special css features. 

I would suggest you have another look on [mobile-first methodology](https://www.invisionapp.com/inside-design/mobile-first-design/) and how it is achieved using media queries. Also use online[ html5 validator](https://validator.w3.org/) to make sure your code is valid.

Also big thumbs up for formatting and keeping your code clean. Again well done!
🙌🌟💯

File: style.css [hunk 5]
```diff
@@ -1 +1,401 @@
...
+  
+.fa-twitter {
+    background: #55acee;
+}
+  
+.fa-instagram {
+    background: #a5678e;
+}
+
+@media (max-width: 539px) {
```
[the remark this answers is not in the document]
Comment (Laura-TM) [reply style.css]: These changes have now been applied

File: style.css (hunk 4 above)
Comment (Laura-TM) [reply style.css]: thank you for your positive comments and your constructive feedback :-)
